src/java.base/share/lib/security/default.policy

```diff
@@ -103,6 +103,19 @@ grant codeBase "jrt:/jdk.accessibility" {
     permission java.lang.RuntimePermission "accessClassInPackage.sun.awt";
 };
 
+grant codeBase "jrt:/jdk.attach" {
+    permission java.lang.RuntimePermission "accessClassInPackage.com.ibm.tools.attach.target";
+    permission java.lang.RuntimePermission "accessClassInPackage.com.ibm.oti.util";
```

Comment:
Please move this line about so the packages are in alphabetical order.

Reply:
Moved up the line in question. 
Please review.